pywb/warcserver/warcserver.py

```diff
@@ -70,6 +73,15 @@ def __init__(self, config_file='./config.yaml', custom_config=None):
 
         self.rules_file = self.config.get('rules_file', '')
 
+        if 'certificates' in self.config:
+            certs_config = self.config['certificates']
+            DefaultAdapters.live_adapter = PywbHttpAdapter(max_retries=Retry(3),
+                                                           cert_reqs=certs_config.get('cert_reqs', 'CERT_NONE'),
+                                                           ca_cert_dir=certs_config.get('ca_cert_dir', '/etc/ssl/certs'))
```

Comment:
```suggestion
                                                           ca_cert_dir=certs_config.get('ca_cert_dir'))
```

Want to make this optional.